### ISSUE TITLE
how to support multiple players?

### ISSUE THREAD

**Issue by bboyazeal** [kedoska/engine-blackjack#19]
can i use this engine to support multiple players?

**Comment by kedoska** [comment]
TL;DR: not now. @bboyazeal the life circle in the engine is not considering multiple players. Right now it assumes only one player against the dealer.

## but
it is certain possible to make it work with more than one player by adding 2 key features:
1. while the stage is `READY` we can allow action like `JOIN_TABLE`. This will populate an `array` of available players before to start the life circle.
2. scale the current `available_actions` by adding the concept of `pending players` before dispatching the `SHOWDOWN`

The main issue is still the coordination (`clock`), but that will remains a task for the server.

## conclusion 
I'm not interested in changing this project to become multi-player but, in a separate project, if you have time, I can probably participate in the development of the components to make a multiplayer blackjack server.
